examples/encoding/live/live_to_vod.py

```diff
@@ -0,0 +1,189 @@
+import datetime
+from bitmovin import Bitmovin, S3Output, EncodingOutput, ACLEntry, ACLPermission, DashManifest, FMP4Representation, \
+    FMP4RepresentationType, Period, VideoAdaptationSet, AudioAdaptationSet, HlsManifest, AudioMedia, VariantStream
+
+# IMPORTANT: first run start_live_encoding_dash_hls.py to get the ids
+
+API_KEY = '<YOUR_API_KEY>'
+
+S3_OUTPUT_ACCESS_KEY = '<YOUR_S3_ACCESS_KEY>'
+S3_OUTPUT_SECRET_KEY = '<YOUR_S3_SECRET_KEY>'
+S3_OUTPUT_BUCKET_NAME = '<YOUR_S3_BUCKET_NAME>'
+
+date_component = str(datetime.datetime.now()).replace(' ', '_').replace(':', '-').split('.')[0].replace('_', '__')
+OUTPUT_BASE_PATH = 'your/output/base/path/{}/'.format(date_component)
+
+# INPUT INFORMATION FROM LIVE STREAM
+ENCODING_ID = 'COPY AND PASTE'
+
+STREAM_1080P_ID = 'COPY AND PASTE'
+STREAM_720P_ID = 'COPY AND PASTE'
+STREAM_480P_ID = 'COPY AND PASTE'
+STREAM_360P_ID = 'COPY AND PASTE'
+STREAM_240P_ID = 'COPY AND PASTE'
+STREAM_AUDIO_ID = 'COPY AND PASTE'
+
+FMP4_MUXING_1080P_ID = 'COPY AND PASTE'
+FMP4_MUXING_720P_ID = 'COPY AND PASTE'
+FMP4_MUXING_480P_ID = 'COPY AND PASTE'
+FMP4_MUXING_360P_ID = 'COPY AND PASTE'
+FMP4_MUXING_240P_ID = 'COPY AND PASTE'
+FMP4_MUXING_AUDIO_ID = 'COPY AND PASTE'
+
+TS_MUXING_1080P_ID = 'COPY AND PASTE'
+TS_MUXING_720P_ID = 'COPY AND PASTE'
+TS_MUXING_480P_ID = 'COPY AND PASTE'
+TS_MUXING_360P_ID = 'COPY AND PASTE'
+TS_MUXING_240P_ID = 'COPY AND PASTE'
+TS_MUXING_AUDIO_ID = 'COPY AND PASTE'
+
+# set the start segment number. If it is set to None the first segment is taken as start segment
+START_SEGMENT = None
```

Comment:
There should also be a setting for END_SEGMENT?

Reply:
END_SEGMENT is not available in the bitmovin-python yet, but I can add and push to this same Pull Request.

Reply:
@msmole done.